tracer/test/snapshots/Security.AspNetCore5AsmActionsConfiguration.__type=block_request_statusCode=200_argument=dummy_rule_actionName=block.verified.txt

```diff
@@ -18,7 +18,7 @@
       http.request.headers.user-agent: Mistake Not...,
       http.request.headers.x-forwarded-for: 86.242.244.246,
       http.response.headers.content-type: application/json,
-      http.status_code: 403,
+      http.status_code: 404,
```

Comment:
Are these changes from `403` to `404` expected? 🤔

Reply:
Yes, when we start this test, we set now the status code to 404 and then switch to 200.